source/administration/object-management/object-delete.rst

```diff
@@ -0,0 +1,176 @@
+.. _minio-object-delete:
+
+===============
+Object Deletion
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+This page summarizes how a ``DELETE`` operation affects objects depending on the configuration of the bucket that contains the object.
+
+Any combination of the following factors may impact how ``DELETE`` operations function:
+
+- :ref:`Bucket versioning <minio-bucket-versioning>`
+- :ref:`Object locking rules <minio-object-locking>`
+- :ref:`Object Lifecycle Management rules <minio-lifecycle-management>`
+- :ref:`Object tiering <minio-lifecycle-management-tiering>`
+- :ref:`Site <minio-site-replication-overview>` or :ref:`bucket <minio-replication-behavior-delete>` replication
+- :ref:`Scanner <minio-concepts-scanner>`
+
+Permissions
+-----------
+
+MinIO uses a :ref:`policy based access control <minio-policy>` system for access management.
+The user or service account must provide the correct policy action and conditions to allow a ``DELETE`` for the bucket and object.
+
+Unversioned Objects
+-------------------
+
+When performing a ``DELETE`` operation on an object in a bucket that does not have versioning enabled, the operation is straightforward.
+After verifying the user or service account has permission to perform the ``DELETE`` operation, MinIO permanently removes the object.
+
+The user or service account requesting the delete action the action must have the :policy-action:`s3:DeleteObject` action permission for the bucket and object.
+
+Versioned Objects
+-----------------
+
+``DELETE`` operations work differently when an object is versioned.
+
+The user or service account must have the :policy-action:`s3:DeleteObjectVersion` action permission for the bucket and object.
+
+Delete operations on the current version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``DELETE`` operation on a versioned object that does not specify a version UUID results in a ``DeleteMarker`` placed as the ``head`` of the object.
+
+In this scenario, MinIO does not actually remove the object or any of its versions from the disk.
+All existing versions of the object remain available to access by specifying the version's UUID.
+When a ``DeleteMarker`` is the head for the object, MinIO does not serve the object for ``GET`` requests that do not specify a version number.
```

Comment:
Yet another `version UUID` is a bit repetitive, but maybe `version ID` would work here
```suggestion
When a ``DeleteMarker`` is the head for the object, MinIO does not serve the object for ``GET`` requests that do not specify a version ID.
```

Reply:
Yes, I'll take this. Thanks!